README.md

```diff
@@ -41,11 +41,10 @@ _MenuList_ extension method has three optional parameters:
 
 Below is an example how menu can be created for pages under start page.
 
-    @helper MenuItemTemplate(MenuItem item)
-    {
-        <li class="@(item.Selected ? "active" : null)">
+    @{
+        Func<MenuItem, HelperResult> menuItemTemplate =@<li class="@(item.Selected ? "active" : null)">
```

Comment:
Could it be possible to write it as a local method? Just curious

```suggestion
        HelperResult GetMenuItemTemplate(MenuItem menuItem) {
            return @<li class="@(item.Selected ? "active" : null)">
                @Html.PageLink(item.Page)
            </li>;
```

Reply:
Doesn't look like it:
![image](https://user-images.githubusercontent.com/1263284/151780902-05298d4b-fc7c-4bdd-9380-e29db9d09e2c.png)

Only something like this but don't think it's particularly useful here:
```csharp
void RenderMenuItemTemplate(MenuItem menuItem)
{
    <li class="@(menuItem.Selected ? "active" : null)">
            @Html.PageLink(menuItem.Page)
    </li>
}
```

Reply:
Seems they do some rewriting magic:
![image](https://user-images.githubusercontent.com/1263284/151781336-f7f59120-f7e6-4ce9-9f94-45ae070af687.png)